# Copyright 2007 Sun Microsystems, Inc.  All rights reserved.
# Use subject to license terms.
#
# Permission is hereby granted, free of charge, to any person obtaining a
# copy of this software and associated documentation files (the
# "Software"), to deal in the Software without restriction, including
# without limitation the rights to use, copy, modify, merge, publish,
# distribute, and/or sell copies of the Software, and to permit persons
# to whom the Software is furnished to do so, provided that the above
# copyright notice(s) and this permission notice appear in all copies of
# the Software and that both the above copyright notice(s) and this
# permission notice appear in supporting documentation.
# 
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
# OF THIRD PARTY RIGHTS. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
# HOLDERS INCLUDED IN THIS NOTICE BE LIABLE FOR ANY CLAIM, OR ANY SPECIAL
# INDIRECT OR CONSEQUENTIAL DAMAGES, OR ANY DAMAGES WHATSOEVER RESULTING
# FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
# NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION
# WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
# 
# Except as contained in this notice, the name of a copyright holder
# shall not be used in advertising or otherwise to promote the sale, use
# or other dealings in this Software without prior written authorization
# of the copyright holder.

Remove -lz from libraries line since Solaris shared libraries already take
care of that dependency.  (Bug 6593503)

Add -I ${includedir} to pkg-config file to fix include paths.  (Bug 6307947)

--- builds/unix/freetype-config.in	2005-06-04 14:58:48.000000000 -0700
+++ builds/unix/freetype-config.in	2007-08-16 15:53:40.933280000 -0700
@@ -141,7 +141,7 @@
   if test "$enable_shared" = "yes" ; then
     eval "rpath=\"$hardcode_libdir_flag_spec\""
   fi
-  libs="-lfreetype @LIBZ@"
+  libs="-lfreetype"
   if test "$libdir" != "/usr/lib" && test "$libdir" != "/usr/lib64"; then
     echo -L$libdir $rpath $libs
   else

--- builds/unix/freetype2.in	2003-06-01 14:30:03.000000000 -0700
+++ builds/unix/freetype2.in	2007-08-16 15:57:00.539973000 -0700
@@ -7,5 +7,5 @@
 Description: A free, high-quality, and portable font engine.
 Version: @ft_version@
 Requires:
-Libs: -L${libdir} -lfreetype @LIBZ@
-Cflags: -I${includedir}/freetype2
+Libs: -L${libdir} -lfreetype
+Cflags: -I${includedir} -I${includedir}/freetype2
